src/plugins/unified_search/public/query_string_input/query_bar_menu.tsx

```diff
@@ -149,7 +175,7 @@ export function QueryBarMenu({
         isOpen={openQueryBarMenu}
         closePopover={closePopover}
         panelPaddingSize="none"
-        anchorPosition="rightUp"
+        anchorPosition="downLeft"
```

Comment:
I went ahead and changed the anchorPosition of the filter menu like I mentioned in the other PR.

Reply:
Yes much better now 🙌